src/libs/Navigation/linkTo.js

```diff
@@ -59,19 +60,23 @@ export default function linkTo(navigation, path, type) {
     const action = getActionFromState(state, linkingConfig.config);
 
     // If action type is different than NAVIGATE we can't change it to the PUSH safely
-    if (action.type === 'NAVIGATE') {
-        // If this action is navigating to the report screen and the top most navigator is different from the one we want to navigate - PUSH
-        if (action.payload.name === NAVIGATORS.CENTRAL_PANE_NAVIGATOR && getTopmostReportId(root.getState()) !== getTopmostReportId(state)) {
-            action.type = 'PUSH';
+    if (action.type === CONST.NAVIGATION.ACTION_TYPE.NAVIGATE) {
+        // In case if type is 'FORCED_UP' we ensure that we need to replace current screen to the provided
```

Comment:
```suggestion
        // In case if type is 'FORCED_UP' we replace current screen with the provided. This means the current screen no longer exists in the stack
```

Reply:
Just trying to clarify that doing this replaces the current screen on the stack so that you can't go back to it.